swagger.yaml

```diff
@@ -39,27 +50,56 @@ parameters:
     description: If true, JSON response will be pretty-printed
 
 definitions:
-  Info:
+  response:
+    type: object
+    properties:
+      links:
+        $ref: "#/definitions/Links"
+      data:
+        $ref: "#/definitions/UserData"
+
+  Links:
+    type: object
+
+  UserData:
```

Comment:
`UserData` doesn't sound right since we're not returning "user data" for this API.